### ISSUE TITLE
New method to support v2 client

### ISSUE THREAD

**Issue by danielpsf** [guregu/dynamo#97]
`dynamo.NewFromIFace(....` could have a sibling to support V2 AWS SDK DDB Client. 😉

**Comment by guregu** [comment]
True, but then I'd have to rewrite the entire library to use v2 OR v1 stuff, and depend on them both.

This sounds crazy, but what you could do is make a translation layer that implements the v1 interface but uses the v2 SDK. Then you could use NewFromIFace with that. It wouldn't need to be part of this package either. I could take a crack at it if I find myself with some extra time, but I'm pretty busy at the moment.

**Comment by magnetikonline** [comment]
Related: https://github.com/guregu/dynamo/pull/206
